input_file.rb

```diff
@@ -0,0 +1,61 @@
+require_relative("sales_tax.rb")
+require_relative("print_receipt.rb")
+
+PRODUCTS_ARR = []
+EXEMPT = %w[chocolates book pills chocolate]
+
+class Entry
+  def input
+    no = 0
+    puts "Number of products?"
+    no = gets.chomp.to_i
+    while no > 0
+      data = gets.chomp
+      split_data = data.split
+      qty = split_data[0].to_i
+      price = split_data[-1].to_f
+      p = data.split(" at ")
+      product1 = p[0].delete("/0-9/")
+      prd = product1.strip
+      product = product1.strip
+      exempt = false
+      imported = false
+      
+      if price <= 0
+        puts "Please enter valid value"
+        break
+      end
+      if qty <= 0
+        puts "Please enter valid value"
+        break
+      end
+      if product.include?('imported')
+        imported = true
+      end
+      chck_product = product.slice! "imported".strip
+      product = product.strip
+      EXEMPT.each do |px|
+        if product.include?(px)
+          exempt = true
+          break
+        end
+        puts px
+      end
```

Comment:
`exempt = (product.split & EXEMPT).present?` Will this work?

Reply:
It's giving error, not working

Reply:
What error?

Reply:
`present` It's not available in ruby, so it's throwing error